website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
 
 ![Welcome Page](</img/image_(1)_(1).png>)
 
-:::tip
-Congratulations! Your Appsmith server should be up and running now. You can access it at [http://localhost](http://localhost).
-:::
-
-### Updating Appsmith (with docker-compose)
+### Updating Appsmith
 
-To update Appsmith (configured with docker-compose) manually, go to the root directory of your setup and run the following command:
+To update Appsmith (configured with docker-compose) manually, go to the root directory of your installation and run the following command:
 
 ```
 docker-compose pull && docker-compose rm -fsv appsmith && docker-compose up -d
 ```
 
-### Enabling Appsmith Auto-updates (with docker-compose)
+### Enabling Appsmith auto-updates
 
-If your Appsmith setup does not have auto-update enabled (i.e. it will not have the Watchtower container running along with Appsmith in the host machine). You can enable auto-update by following the following steps:
+Automatic updates are disabled by default in the docker-compose.yml file. If you would like to enable automatic updates for Appsmith, please follow these steps:
 
-1. Go to the root directory of your Appsmith set up and run:
+1. Go to the root directory of your Appsmith instance and run:
 
 ```
 docker-compose down
 ```
 
-1. Open the `docker-compose.yml` file with any text editor and uncomment all the lines that are commented out (lines 13-23).
-2. Run the command:
+2. Open the `docker-compose.yml` file with any text editor and uncomment all the lines that are commented out (lines 13-23).
+3. Save the file and run:
 
 ```
 docker-compose up -d
 ```
 
-## Explore Appsmith (without docker-compose)
+:::info
+To complement automatic updates, Appsmith also has an auto-backup feature. Before an automatic update, Appsmith first creates a backup to ensure you can [roll back](/getting-started/setup/instance-management/appsmithctl#restore-appsmith-instance) an update to a previous version of Appsmith if necessary. For this auto-backup to work, please ensure that there is a minimum of 2 GB of free storage; otherwise, both the backup and update tasks fail.
+:::
 
-To quickly get Appsmith up and running, run the following command on your machine:
+This configuration runs an Appsmith instance and a Watchtower instance to keep Appsmith automatically up-to-date. Your installation is now configured to stay up-to-date automatically.
 
-### Community Edition
+---
 
-```bash
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" --pull always appsmith/appsmith-ce
-```
+## Setup with Docker run
 
-### Business Edition
+To get Appsmith up and running, run the following command on your machine:
 
-```
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" -e APPSMITH_LICENSE_KEY=<Add_YOUR_LICENSE_KEY> --pull always appsmith/appsmith-ee
-```
+### Community Edition setup
```

Comment:
remove setup from here as it is already mentioned in the main heading. Do the same for Business.